# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/slmodem/nodepmod.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN slmodem-2.9.8/drivers/Makefile slmodem-2.9.8-new/drivers/Makefile
--- slmodem-2.9.8/drivers/Makefile	2003-12-21 22:15:54.000000000 +0100
+++ slmodem-2.9.8-new/drivers/Makefile	2004-06-24 15:52:10.855671664 +0200
@@ -75,14 +75,14 @@
 	cp /etc/modules.conf /etc/modules.conf.slamr && \
 	    echo 'alias char-major-212 slamr' >> /etc/modules.conf && \
 	    echo 'alias char-major-213 slusb' >> /etc/modules.conf 
-	/sbin/depmod -a
+	echo 'ROCK: Please run /etc/cron.d/00-updates afterwards!'
 uninstall:
 	/sbin/modprobe -r slamr slusb
 	cp /etc/modules.conf /etc/modules.conf.slamr && \
 	   egrep -ve 'alias char-major-212 slamr|alias char-major-213 slusb' /etc/modules.conf.slamr > /etc/modules.conf
 	$(RM) $(module-dir)/slamr.o
 	$(RM) $(module-dir)/slusb.o
-	/sbin/depmod -a
+	echo 'ROCK: Please run /etc/cron.d/00-updates afterwards!'
 uninstall-old:
 	$(RM) $(module-dir)/slmdm.o \
               $(module-dir)/slfax.o \
@@ -102,13 +102,13 @@
 install:
 	install -D -m 644 slamr.ko $(module-dir)/slamr.ko
 	install -D -m 644 slusb.ko $(module-dir)/slusb.ko
-	/sbin/depmod -a
+	echo 'ROCK: Please run /etc/cron.d/00-updates afterwards!'
 uninstall:
 	modprobe -r slamr ; echo -n
 	modprobe -r slusb ; echo -n
 	$(RM) $(module-dir)/slamr.ko
 	$(RM) $(module-dir)/slusb.ko
-	/sbin/depmod -a
+	echo 'ROCK: Please run /etc/cron.d/00-updates afterwards!'
 endif
 
 endif
